tests/integration/security/util/rbac_util/util.go

```diff
@@ -33,6 +34,10 @@ type TestCase struct {
 	ExpectAllowed bool
 	Jwt           string
 	Headers       map[string]string
+	// Indicates whether a test should be run in the multicluster environment.
+	// This is a temporary flag during the converting tests into multicluster supported.
+	// TODO: Remove this flag when all tests support multicluster
+	SkippedForMulticluster bool
```

Comment:
Maybe better to have this be `Skip string`. If the string isn't `""` then `ctx.Skip(tc.Skip)`. More robust to things besides multicluster. 

The behavior of this as it sits is sort of confusing, it just skips the one part of validation rather than a bunch of testing that we wouldn't need to run anyway.